### DISCUSSION TITLE
My student GitHub Pro was unexpectedly removed

### DISCUSSION THREAD

**Discussion by Dannomac** [community/community#44263] [GitHub Education]
Hello,
I had a case open with support, but it was closed. The letter I used to verify my student account is supposed to last until I leave school, but my GitHub Pro has been removed. I am unable to reapply using the same letter.

**Comment by hanminshin** [comment]
> Hello, I had a case open with support, but it was closed. The letter I used to verify my student account is supposed to last until I leave school, but my GitHub Pro has been removed. I am unable to reapply using the same letter.

fake account

**Comment by samus-aran** [comment]
Hi @Dannomac 👋 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. Students are asked to reverify themselves between 2-3 years or may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.
To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.

**Reply by Dannomac** [reply]
Hello,
I completely understand. In Ireland, we don't do student IDs, so I had to get a letter from my year head. The contents of the letter verify my student status for a few more years. I am unable to reuse the same letter again on the portal.

**Comment by Dannomac** [comment]
Hello,
I completely understand. In Ireland, we don't do student IDs, so I had to get a letter from my year head. The contents of the letter verify my student status for a few more years. I am unable to reuse the same letter again on the portal.

**Comment by Dannomac** [comment]
@samus-aran
Hello,
Sorry for being impatient. Is there any update? I would like to get back my GitHub Pro status.

**Reply by samus-aran** [reply]
Hi @Dannomac 👋 

Thank you for your patience. GitHub Global Campus asks for students to reverify every couple of years. We understand that students' length of study can vary. Does your letter state when your enrolment ends? If it doesn't then we would first recommend to reach out to your school (or teachers) and ask them to consider applying for GitHub's School Partners so that the school can identify a preferred form of verification for other students e.g. an academic school email. This may assist you with providing another form of evidence as your previous evidence has already been submitted.

**Reply by Dannomac** [reply]
@samus-aran 
Thanks for the quick reply. While the letter does not have a specific date on it, it does specify the date when I started in my school and that I will be there until I finish my post-primary education (which in Ireland is 6 years). I am currently out sick at the moment, so if this is not enough, I will have to wait until next week to get another letter.

**Reply by Dannomac** [reply]
@samus-aran 
Is there any update available? I am currently out with COVID so if I have to get a new letter, I'll have to wait until I'm out of self-isolation.
